README.md

```diff
@@ -9,7 +9,7 @@ Parallel and distributed execution currently is MPI/CSP-like. In a later version
 
 ## Setting up build environment
 
-Install MLIR/LLVM and Intel® Extension for MLIR (IMEX, see https://github.com/intel/mlir-extensions). Besides, Add "-DLLVM_ENABLE_RTTI=ON" into cmake arguments, and build target is "all" (not "check-imex").
+Install MLIR/LLVM and Intel® Extension for MLIR (IMEX, see https://github.com/intel/mlir-extensions). Besides, Add `-DLLVM_ENABLE_RTTI=ON` into cmake arguments, and build target is `all` (not `check-imex`).
```

Comment:
..... Make sure you use `-DLLVM_ENABLE_RTTI=ON` when configuring LLVM and use build build target `all`.

Reply:
Nice! Done.